extractor/src/main/java/org/schabi/newpipe/extractor/stream/AudioStream.java

```diff
@@ -421,15 +485,36 @@ public String getAudioTrackId() {
     }
 
     /**
-     * Get the name of the audio track.
+     * Get the name of the audio track, which may be {@code null} if this information is not
+     * provided by the service.
      *
-     * @return the name of the audio track
+     * @return the name of the audio track or {@code null}
      */
     @Nullable
     public String getAudioTrackName() {
         return audioTrackName;
     }
 
+    /**
+     * Get the {@link Locale} of the audio representing the language of the stream, which is
+     * {@code null} if the audio language of this stream is not known.
+     *
+     * @return the {@link Locale} of the audio or {@code null}
+     */
+    @Nullable
+    public Locale getAudioLocale() {
+        return audioLocale;
+    }
+
+    /**
+     * Returns whether this stream is a descriptive audio.
```

Comment:
Since this is the API accessible from library users, I think it's best to document it here, too.
(also, no need to write "Returns ..." and then "@return ..." twice)

```suggestion
     * A descriptive audio is an audio in which descriptions of visual elements of a video are
     * added in the original audio, with the goal to make a video more accessible to blind and
     * visually impaired people.
     *
```

Reply:
> also, no need to write "Returns ..." and then `@return ...` twice

I agree if there is only the "return" text. Now that I've added the description in the Javadoc for the getter, I think starting the doc with the description would be strange, so I kept it.

I also changed the `return` Javadoc description to give an explicit return value.